src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonInfo } from './PersonInfo';
+
+export const PeopleTable:React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isActive, setIsActive] = useState('');
```

Comment:
```suggestion
  const [activePerson, setActivePerson] = useState('');
```
It's not boolean value, so `is` is redundant here